Examples/Physics_applications/thomson_parabola_spectrometer/inputs

```diff
@@ -0,0 +1,195 @@
+
+carbon12_6.charge = 6*q_e
+carbon12_6.initialize_self_fields = 0
+carbon12_6.injection_style = gaussian_beam
+carbon12_6.mass = 12*m_p
+carbon12_6.momentum_distribution_type = uniform
+carbon12_6.npart = N_macro_particles
+carbon12_6.q_tot = N_real_particles*6*q_e
+carbon12_6.ux_min = 0
+carbon12_6.uy_min = 0
+carbon12_6.uz_min = sqrt(Emax_carbon12_6/(12*m_p))/clight
+carbon12_6.ux_max = 0
+carbon12_6.uy_max = 0
+carbon12_6.uz_max = sqrt(2*Emax_carbon12_6/(12*m_p))/clight
+carbon12_6.x_m = 0
+carbon12_6.x_rms = 0
+carbon12_6.y_m = 0
+carbon12_6.y_rms = 0
+carbon12_6.z_m = 0
+carbon12_6.z_rms = 0
+carbon12_6.do_not_gather = 1
+carbon12_6.do_not_deposit = 1
+
+carbon12_4.charge = 4*q_e
+carbon12_4.initialize_self_fields = 0
+carbon12_4.injection_style = gaussian_beam
+carbon12_4.mass = 12*m_p
+carbon12_4.momentum_distribution_type = uniform
+carbon12_4.npart = N_macro_particles
+carbon12_4.q_tot = N_real_particles*4*q_e
+carbon12_4.ux_min = 0
+carbon12_4.uy_min = 0
+carbon12_4.uz_min = sqrt(Emax_carbon12_4/(12*m_p))/clight
+carbon12_4.ux_max = 0
+carbon12_4.uy_max = 0
+carbon12_4.uz_max = sqrt(2*Emax_carbon12_4/(12*m_p))/clight
+carbon12_4.x_m = 0
+carbon12_4.x_rms = 0
+carbon12_4.y_m = 0
+carbon12_4.y_rms = 0
+carbon12_4.z_m = 0
+carbon12_4.z_rms = 0
+carbon12_4.do_not_gather = 1
+carbon12_4.do_not_deposit = 1
+
+###########
+#### FIELDS
+###########
+particles.E_ext_particle_init_style = parse_E_ext_particle_function
+particles.Ex_external_particle_function(x,y,z,t) = "E0*(z>d1)*(z<(d1+d2))"
+particles.Ey_external_particle_function(x,y,z,t) = 0
+particles.Ez_external_particle_function(x,y,z,t) = 0
+
+particles.B_ext_particle_init_style = parse_B_ext_particle_function
+particles.Bx_external_particle_function(x,y,z,t) = "B0*(z>d1+d2)*(z<(d1+d2+d3))"
+particles.By_external_particle_function(x,y,z,t) = 0
+particles.Bz_external_particle_function(x,y,z,t) = 0
+
+################
+#### DIAGNOSTICS
+################
+diagnostics.diags_names = diag0 screen diag1
+
+diag0.diag_type = Full
+diag0.fields_to_plot = none
+diag0.format = openpmd
+diag0.intervals = 0:0
+diag0.openpmd_backend = bp
+diag0.write_species = 1
+diag0.species = hydrogen1_1 carbon12_6 carbon12_4
+diag0.dump_last_timestep = 0
+
+# diagnostic that collects the particles at the detector's position,
+# i.e. when a particle exits the domain from z_max = zhi
+# we store it in the screen diagnostic
+# we are assuming that most particles will exit the domain at z_max
+# which requires a large enough transverse box
+screen.diag_type = BoundaryScraping
+screen.format = openpmd
+screen.intervals = 1
+screen.openpmd_backend = bp
+hydrogen1_1.save_particles_at_zhi = 1
+carbon12_6.save_particles_at_zhi = 1
+carbon12_4.save_particles_at_zhi = 1
+
+diag1.diag_type = Full
+diag1.fields_to_plot = Ex Bx
+diag1.format = openpmd
+diag1.intervals = 100
+diag1.openpmd_backend = bp
+diag1.write_species = 1
+diag1.species = hydrogen1_1 carbon12_6 carbon12_4
+diag1.dump_last_timestep = 1
```

Comment:
`diag1` plots the simulation results also at timestep 0, which is what `diag0` does. Is this the intended behavior?